assets/js/src/form_editor/components/history_redo.tsx

```diff
@@ -3,21 +3,47 @@ import { __ } from '@wordpress/i18n';
 import { Button } from '@wordpress/components';
 import { useSelect, useDispatch } from '@wordpress/data';
 import { redo as redoIcon } from '@wordpress/icons';
+import { displayShortcut } from '@wordpress/keycodes';
+import { useShortcut } from '@wordpress/keyboard-shortcuts';
 
 function HistoryRedo(props) {
   const hasRedo = useSelect(
     (select) => select('mailpoet-form-editor').hasEditorRedo(),
     []
   );
   const { historyRedo } = useDispatch('mailpoet-form-editor');
+  const { registerShortcut } = useDispatch('core/keyboard-shortcuts');
+
   const redoAction = () => {
     historyRedo();
   };
+
+  useShortcut(
+    // Shortcut name
+    'mailpoet-form-editor/redo',
+    // Shortcut callback
+    (event) => {
+      redoAction();
+      event.preventDefault();
+    },
+  );
+
+  registerShortcut({
+    name: 'mailpoet-form-editor/redo',
+    category: 'block',
+    description: __('Redo your last undo.'),
```

Comment:
This is using the gutenberg default translation function. Is that intentional?

Reply:
Yes, both translations are from Gutenberg.